docs/Snackbar/README.md

```diff
@@ -0,0 +1,30 @@
+# Snackbar
+
+## Installation
+
+Backpack Android is available through [Jitpack](https://jitpack.io/#Skyscanner/backpack-android). Check the main [Readme](https://github.com/skyscanner/backpack-android#installation) for a complete installation guide.
+
+## Usage
+
+The Snackbar uses builder pattern and can be used in Kotlin/Java
+
+```Kotlin
+import net.skyscanner.backpack.snackbar.BpkSnackbar
+
+BpkSnackbar.builder(view, "Message", BpkSnackbar.LENGTH_SHORT)
+        .setAction("Action") {}
+        .build()
+        .show()
+```
+It's strongly not recommended to change message or action once the Snackbar has been built.
```

Comment:
```suggestion
It's strongly not recommended not to change the message or action after the Snackbar has been built. This might result in losing the themed attributes
```

Reply:
Done